action.yml

```diff
@@ -8,6 +8,7 @@ inputs:
    token:
       description: GitHub token. Required only if 'version' == 'latest'
       required: false
+      default: "${{ github.token }}"
```

Comment:
is defaulting to reading the github token here necessary since it's only used for the `latest` case? opt-in to token use would keep the token to where it's absolutely needed

Reply:
Just replicated from approved and staled #112